---
float32[6] setangle